README.md

```diff
@@ -12,9 +12,12 @@ Execute the following installation steps to use Mapago:
 ```
 export GOPATH=$HOME/go
 export GOBIN=$HOME/go/bin
+export GOROOT=/usr/local/go
```

Comment:
no, I have no access right. This required ROOT! No way. Why the GOROOT now? This is far to many env variables. With current go there is no need for such things!